spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -1,190 +1,150 @@
 require 'spec_helper'
 
-shared_examples_for 'GET show success' do
-  it { is_expected.to respond_with(:success) }
-  it { is_expected.to render_template(:show) }
-  it { is_expected.not_to set_flash }
-  it 'should set to correct equipment model' do
-    expect(assigns(:equipment_model)).to eq(model)
-  end
-  it 'should set @associated_equipment_models' do
-    mod1 = FactoryGirl.create(:equipment_model)
-    model.associated_equipment_models = [mod1]
-    get :show, id: model
-    expect(assigns(:associated_equipment_models).size).to eq(1)
-    expect(assigns(:associated_equipment_models)).to eq([] << mod1)
-  end
+describe EquipmentModelsController, type: :controller do
+  before(:each) { mock_app_config }
+  let!(:model) { FactoryGirl.create(:equipment_model) }
 
-  it 'should limit @associated_equipment_models to maximum 6' do
-    mod1 = FactoryGirl.create(:equipment_model)
-    mod2 = FactoryGirl.create(:equipment_model)
-    mod3 = FactoryGirl.create(:equipment_model)
-    mod4 = FactoryGirl.create(:equipment_model)
-    mod5 = FactoryGirl.create(:equipment_model)
-    mod6 = FactoryGirl.create(:equipment_model)
-    mod7 = FactoryGirl.create(:equipment_model)
-    model.associated_equipment_models =
-      [mod1, mod2, mod3, mod4, mod5, mod6, mod7]
-    get :show, id: model
-    expect(assigns(:associated_equipment_models).size).to eq(6)
+  it_behaves_like 'calendarable', EquipmentModel
+
+  USER_ROLES = [:admin, :user].freeze
+
```

Comment:
the new `shared_examples` should probably live in a separate file where it can be accessed by all controller specs